source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,135 +4,382 @@ Sort Results
-    import static com.mongodb.client.model.Sorts.orderBy;
 
-    // sort first by length ascending, then title descending
-    MongoCursor<Document> cursor = collection.find()
-        .sort(orderBy(ascending("length"), descending("title")))
-        .iterator();
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter"));
+
+Since multiple documents contain "a" on the field we are sorting on, the first
+document returned could be any of the following documents:
+
+.. code-block:: json
 
-In the returned documents, notice how the two books with the same 
-length - "Infinite Jest" and "A Dance with Dragons" -
-appear in the reverse of the order shown in the previous example:
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+If you need guaranteed sort order for documents that
+have fields with identical values, you can specify additional fields to sort
+on in the event of a tie.
+
+We can specify an ascending sort on the ``letter`` field followed by the
+``_id`` field as follows:
 
 .. code-block:: java
 
-    Document{{_id=1, title=The Brothers Karamazov, length=824, author=Dostoyevksy}}
-    Document{{_id=5, title=Cryptonomicon, length=918, author=Stephenson}}
-    Document{{_id=3, title=Atlas Shrugged, length=1088, author=Rand}}
-    Document{{_id=4, title=Infinite Jest, length=1104, author=Wallace}}
-    Document{{_id=6, title=A Dance with Dragons, length=1104, author=Martin}}
-    Document{{_id=2, title=Les Misérables, length=1462, author=Hugo}}
+   import static com.mongodb.client.model.Sorts.ascending;
 
-For additional information on the classes and methods mentioned on this
-page, see the following API documentation:
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"));
 
-- :java-sync-api:`FindIterable <com/mongodb/client/FindIterable.html>`
-- :java-sync-api:`MongoCursor <com/mongodb/client/MongoCursor.html>`
-- :java-docs:`Sorts </apidocs/mongodb-driver-core/com/mongodb/client/model/Sorts.html>`
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-crud-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+Combining Sort Criteria
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. The ``orderBy()`` method builds sort criteria that apply passed 
+in sort criteria from left to right in the event of ties. 
```

Comment:
1) The first two sentences refer to the method two different ways which might cause some disconnect for the reader. Instead, we can use a pronoun to avoid repeating the terminology.

2) By "passed in" (prepositional phrase), I think you meant "passed-in" (adjective). However, I don't think this is the greatest adjective to describe parameters. We can omit it and the sentence still communicates the same message.

3) "builds sort criteria that apply passed in sort criteria" doesn't make much sense to me. Are there two different types of sort criteria? How do I distinguish between them?

 
```suggestion
To combine sort criteria, use the ``Sorts.orderBy()`` static factory
method. This method constructs an object containing an ordered list of sort criteria. 
When performing the sort, if the leftmost sort criteria results in a tie, the sort uses the
next sort criteria in the list to determine the order.
```

Reply:
Thank you for the note. I had a lot of trouble with this introduction, and this phrasing is much better than what I had. I think my second sentence was doing too much work, and your breaking it into two sentences significantly improved clarity. 

Going forward, I will look for opportunities to use pronouns to  avoid repeating terminology when it does not cause problems with clarity. 

Going forward, I will try and avoid using the same terms in slightly different ways in the same sentence. A few of your notes have touched on my doing this in different places.